srcpkgs/arpack-ng/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'arpack-ng'
 pkgname=arpack-ng
-version=3.7.0
+version=3.8.0
 revision=1
 build_style=gnu-configure
+configure_args=" --with-blas=openblas --with-lapack=openblas"
```

Comment:
As far as I can see, this will make it not build on mips or ppc (not ppc64) anymore...

Idk if @pullmoll or @q66 care about that, but then it might make sense to clean up the package.

Can you explain here (and in the commit message) the reasoning for switching to openblas?